augur/util_support/date_disambiguator.py

```diff
@@ -149,3 +142,24 @@ def assert_only_less_significant_uncertainty(self):
                 raise InvalidDate(self.uncertain_date,
                     "Month contains uncertainty, so day must also be uncertain."
                 )
+
+    def set_bounds(self, min_max_year):
+        """Set the exact day bounds."""
+        if len(min_max_year) > 2:
+            raise AugurError(f"The year bounds '{min_max_year}' must have only one (lower) or two (lower, upper) bounds.")
+
+        # TODO: Allow more granularity than year alone.
+
+        try:
+            self.lower_bound = datetime.date(int(min_max_year[0]), 1, 1)
+        except:
+            raise AugurError(f"'{min_max_year[0]}' is not a valid year.")
```

Comment:
This interpolation for AugurError will raise an IndexError if `min_max_year` is an empty list/tuple/sequence. I know that's covered by the conditional call to `set_bounds()` in the constructor, but this isn't a private function and we should probably guard against an empty sequence here too.

Reply:
Done: f061d76097dc3d87a5b41d0b424be69dd0c8da8f